tools/rapids-download-docker-from-s3

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# A utility script that downloads a docker image from S3
+# Positional Arguments:
+#   1) image tag
+set -eo pipefail
+source rapids-constants
+export RAPIDS_SCRIPT_NAME="rapids-download-docker-from-s3"
+
+tmp_dir="$(mktemp -d)"
+
+if [ -z "$1" ]; then
+  rapids-echo-stderr "Must specify input argument: IMAGE_TAG"
+  exit 1
+fi
+
+docker_image="$1"
+
+# replace "/..." with "_..." to use as file name
+docker_image_no_slash="${docker_image//\//_}"
+docker_image_s3_name="docker_${docker_image_no_slash}.tar.gz"
+tmp_fname="${tmp_dir}/${docker_image_no_slash}.tar.gz"
+
+# download .tar.gz into tmpdir
+s3_dl_path="$(rapids-s3-path)${docker_image_s3_name}"
```

Comment:
Hmm I guess this bit is different from `rapids-download-from-s3`. Also `rapids-download-from-s3` assumes that the uploaded content is a tarball. In this case I guess we don't want to untar it because `docker load` accepts a `tar.gz` directly? Should we rework `rapids-download-from-s3`, or is that too much work? @ajschmidt8 what do you think?

Reply:
`rapids-download-from-s3` could be stale; I never used it much and it might be in an older/less working state. But, it would still be useful if we were to ever run a separate unit test or security scan layer (and want to download the intermediate containers to test/scan).

Reply:
I don't follow here. It seems like any script has an equal chance of going stale, wouldn't we be better off consolidating so that we have a single point of maintenance and know not to let it go stale?